packages/devtools_app/lib/src/shared/new_issue.dart

```diff
@@ -9,8 +9,8 @@ import 'utils.dart';
 const newDevToolsIssueUriDisplay = 'github.com/flutter/devtools/issues/new';
 
 Uri newDevToolsIssueUri({String? issueDetails}) {
-  final issueBodyItems = _issueLinkDetails();
-  if (issueDetails != null) issueBodyItems.add(issueDetails);
+  final issueBodyItems = issueLinkDetails();
```

Comment:
nit: having the if check inside the list is a little cleaner. This can be written as follows :
```dart
final issueBodyItems = [
  if (issueDetails != null) issueDetails,
  ...issueLinkDetails(),
];
```